packages/amplify_auth_cognito_plugin/amplify_auth_cognito/example/tool/add_api_request.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Coming back to this - I think they are show as added rather than moved because they are duplicated in the _android package. I think the /tool dir can be removed from the _android package.

Reply:
Yup - good catch, thanks.